python/tests/test_pylint.py

```diff
@@ -18,16 +18,17 @@
 
 class LintErt(LintTestCase):
     """Tests that no file in ert needs linting"""
+    pass
```

Comment:
It would be good if we could configure `test_pylint` to accept monkey patching, otherwise  the lint testing will be ineffective for a long time.